PSKoans/Koans/IntroTraining/AboutExamples.koans.ps1

```diff
@@ -0,0 +1,94 @@
+using module PSKoans
+[Koan(Position = 102)]
+param()
+<#
+
+    Some fun examples!
+
+    PowerShell is a super fun, and versitile, scripting language based off of .NET.
+    To get you excited about the language we've given you some sample code to run.
+
+    Once you've gone through the examples simply mark each __ as $true
```

Comment:
🤔 I'm not sure this really needs to be a koan topic in and of itself?

Things like this would be great as maybe little advice snippets for Get-Advice that we have over in the`PSKoans/Data/Advice` folder?

I might have to veto the rickroll one, though; as cool as it is, it's definitely something security-minded folks will raise a stink over. Combination of blindly executing the script and `iex` makes it something I'd be very cautious of ever recommending outright.

However, if we just provide a link for folks to check it out in an advice snippet or something, I think that's more than enough for what it is. It's a great example of what's possible, but I can't imagine the code looks especially great. 😁

Reply:
I think I'll remove the rickroll one; I don't want to introduce it then immediately say "don't click this but do if you want to".

Didn't see the JSON advice files either, they're pretty neat! I've turned the Cats and GUI examples into individual JSON files and removed the koan.

Is it worth mentioning the ```Get-Advice``` cmdlet in the readme or are you going to wait till there's some more JSON files in there?

Reply:
I have been waiting for more advice snippets before I do, but thinking about it... it probably makes more sense to mention it now so that folks can know about it & suggest entries that'd work well with it. 

Might be worth mentioning there for sure. :)